### PR TITLE
Ensnared hostile will now abort the move order

### PR DESCRIPTION
**Pull request by Jaccouille**
From what I saw, creep behavior seems to issue an move order back to initial position after ~6 seconds.
Made small bunch of code which will abort the move order when the unit's being targeted by ensnare ability.
showcase

Also added a "-hostile" command to give selected unit to PLAYER_NEUTRAL_AGGRESSIVE

### CODE REVIEW

File: wurst/systems/commands/SpawningTestHelpers.wurst [hunk 1]
```diff
@@ -46,3 +46,8 @@ init
                                 break
                     else
                         simError(triggerPlayer, "Cannot spawn {0}".format(idStr))
+
+            registerCommandAll("hostile") (triggerPlayer, args) ->
+                let units = CreateGroup()..enumUnitsSelected(triggerPlayer, null)
+                for u in units
+                    u.setOwner(players[PLAYER_NEUTRAL_AGGRESSIVE], true)
```
Comment (theQuazz) [inline wurst/systems/commands/SpawningTestHelpers.wurst]: You need to also call `units.destr()` at the end and/or call `for u from units`

File: wurst/systems/spawns/HostileNetBehavior.wurst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package HostileNetBehavior
+import ClosureTimers
+import RegisterEvents
+import ID
+
+constant let MOVE_ORDER = String2OrderIdBJ("move")
```
Comment (theQuazz) [inline wurst/systems/spawns/HostileNetBehavior.wurst]: You could also use `Orders.smart` from package `Orders`

Comment (Jaccouille) [reply wurst/systems/spawns/HostileNetBehavior.wurst]: /** This is a point or object targeted order that is like a right click. */

I don't really get it, what's the purpose?

Comment (theQuazz) [reply wurst/systems/spawns/HostileNetBehavior.wurst]: Or just `Orders.move`

Comment (Jaccouille) [reply wurst/systems/spawns/HostileNetBehavior.wurst]: Oh yeah I could use that

**Pull request merged**